src/vmspawn/vmspawn.c

```diff
@@ -357,9 +357,23 @@ static int run_virtual_machine(void) {
         if (r < 0)
                 return log_oom();
 
-        r = strv_extend_strv(&cmdline, arg_parameters, false);
-        if (r < 0)
-                return log_oom();
+        if (strv_length(arg_parameters) != 0) {
```

Comment:
!strv_isempty(...)

Reply:
ah I looked for strv_empty but not isempty, thanks :)